wordpress-mobile/WordPress-iOS
# Avoid scroll up down

This PR updates the Aztec controller to use the new version of the pod where refresh of attachment overlays can be done without doing a relayout of the full post.

To test:
 - Create a new Aztec Post
 - Write some test on the top of post
 - Add some image to the post
 - While uploads are going scroll up to the initial text and try to write there
 - Check that no scrolls happens


P.S: there is still some scroll happening when the image thumbnail is first added (attachment changes size) but it should happen no more while progress is ongoing.
